docs-en/chapter_preface/suggestions.md

```diff
@@ -2,223 +2,225 @@
 
 !!! tip
 
-    For the best reading experience, it is recommended that you read through this section.
+    It is recommended that you go through this section to get the best reading experience.
```

Comment:
Hi QiLOL
all of these I'm going to list are pretty optional, you can choose rather to use it or not.
"It is recommended that you go through this section to get the best reading experience." 
can be changed to ->
 "For the best reading experience, it is recommended that you read through this section."